toonz/sources/common/tgeometry/tgeometry.cpp

```diff
@@ -58,14 +58,18 @@ TAffine TAffine::inv() const {
   if (a12 == 0.0 && a21 == 0.0) {
     assert(a11 != 0.0);
     assert(a22 != 0.0);
-    double inv_a11 = 1.0 / a11;
-    double inv_a22 = 1.0 / a22;
+    double inv_a11 =
+        (a11 == 0.0 ? std::numeric_limits<double>::max() / 2 : 1.0 / a11);
+    double inv_a22 =
+        (a22 == 0.0 ? std::numeric_limits<double>::max() / 2 : 1.0 / a22);
     return TAffine(inv_a11, 0, -a13 * inv_a11, 0, inv_a22, -a23 * inv_a22);
```

Comment:
Sorry if I am wrong, but when `inv_a11` is set to `std::numeric_limits<double>::max() / 2` , does the value `-a13 * inv_a11` easily overflow when `a13` is bigger than 2.0 ?